landoapi/secapproval.py

```diff
@@ -86,7 +88,7 @@ def send_sanitized_commit_message_for_review(revision_phid, message, phabclient)
             {"type": "reviewers.add", "value": [f"blocking({sec_approval_phid})"]},
         ],
     )
-    return PhabricatorClient.expect(response, "transactions")
```

Comment:
nit: This change is not just a rename, should probably just include that detail in the commit message.

Reply:
It is included in the commit message.